### PR TITLE
Ana/fix concat of undefined in ClaimRewardsTxDetails

### PR DESCRIPTION
**Pull request by Bitcoinera**
Closes #ISSUE

**Description:**

Another easy check fix. Don't know how I didn't notice while coding this. Probably it was all cached already.

I think it is a good practice to turn the vue server down and then restart to catch this kind of silly bugs.

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/components/transactions/message-view/ClaimRewardsTxDetails.vue [hunk 1]
```diff
@@ -41,7 +41,11 @@
     >
       <div class="tx__content__left multi-claim-reward-row">
         <h3 class="multi-claim-reward-h3">{{ caption }}</h3>
-        <div class="multi-claim-reward-row" :class="{ validatorsToggle: show }">
+        <div
+          v-if="getValidators.length > 0"
```
Comment (faboweb) [inline src/components/transactions/message-view/ClaimRewardsTxDetails.vue]: why is this necessary? line 50 handles an empty array gracefully

Comment (Bitcoinera) [reply src/components/transactions/message-view/ClaimRewardsTxDetails.vue]: You mean line 39? true... I didn't notice. So this is then no the fix probably.

OK, but please don't close this PR as this is still an issue. Will look into it

Comment (Bitcoinera) [reply src/components/transactions/message-view/ClaimRewardsTxDetails.vue]: OK, just fixed it the real way. But we really need to fix API taking too much time to load validators
